search/includes/classes/class-search.php

```diff
@@ -690,7 +690,7 @@ public function filter__ep_bulk_items_per_page() {
 		return 500;
 	}
 
-	public function filter__ep_do_intercept_request( $request, $query, $args ) {
+	public function filter__ep_do_intercept_request( $request, $query, $args, $type ) {
```

Comment:
@rebeccahum Is there a scenario where changing this method's signature may break things?

Reply:
I don't believe so since `$type` is assigned a `null` value by default from `remote_request()`: https://github.com/Automattic/ElasticPress/blob/e623992b60a871c3d6396e35793f217ebb2a5b39/includes/classes/Elasticsearch.php#L1071.

Reply:
@rinatkhaziev I don't believe so since `$type` is passed in as a `null` value by default in `remote_request()` if not specified: https://github.com/Automattic/ElasticPress/blob/e623992b60a871c3d6396e35793f217ebb2a5b39/includes/classes/Elasticsearch.php#L1071.

Reply:
Just to be extra-safe and safe-guard from a potential revert upstream. 
```suggestion
	public function filter__ep_do_intercept_request( $request, $query, $args, $type = ''  ) {
```